app/views/devise/mailer/reset_password_instructions.html.slim

```diff
@@ -1,13 +1,33 @@
-p
+p.lead
   'You have asked #{APP_NAME} to reset your password to log in to your
   '#{APP_NAME} Account. To confirm your request, please click on the link
   'below, or copy and paste the entire link into your browser.
 
+table.button.expanded.large.radius
+  tbody
+    tr
+      td
+        table
+          tbody
+            tr
+              td
+                center
+                  = link_to 'Confirm password reset', \
```

Comment:
I wouldn't use the word 'confirm' in this message.  How about just "Reset password", or "Continue password reset"?